test/e2e/service_to_service_test.go

```diff
@@ -216,5 +221,145 @@ func TestServiceToServiceCallFromZero(t *testing.T) {
 	}
 
 	// Send request to helloworld app via httpproxy service
-	testProxyToHelloworld(t, clients, helloWorld.Route.Status.URL.Host)
+	testProxyToHelloworld(t, clients, helloWorld.Route.Status.URL.Host, nil)
+}
+
+// set up tests with sidecar disabled for both service
+func TestSvcToSvcCallFromZeroWithoutSidecar(t *testing.T) {
```

Comment:
can we extract this test into a common function and create a table test to call them all?  some examples:

https://github.com/knative/serving/blob/master/test/e2e/service_to_service_test.go#L177
https://github.com/knative/serving/blob/master/test/e2e/grpc_test.go#L204

Reply:
Thanks for the comments. Updated code.